modules/microshift-kubeconfig-remote-con.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="remote-access-con_{context}"]
+= Remote access kubeconfig files
+
+Remote access `kubeconfig` file values can be configured. {product-title} generates a default `kubeconfig` for external access using the `hostname` value. An additional `kubeconfig` file is generated for each entry in the `apiServer.subjectAltNames` fields. You can copy these files from the host during times of IP connectivity and then implement them from other workstations.
+
+When a {product-title} cluster connects to the API server from an external source, a certificate with all of the alternative names in the SAN field is used for validation.
+
+.Example certificate for remote access
+[source, terminal]
+----
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 2 (0x2)
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN = kube-apiserver-external-signer
+        Validity
+            Not Before: Mar 17 08:44:07 2023 GMT
+            Not After : Mar 16 08:44:08 2024 GMT
+        Subject: CN = 10.0.0.1
+...
+            X509v3 Subject Alternative Name:
+                DNS:api.example.com, DNS:microshift-rhel9, DNS:alt-name-1, DNS:10.0.0.1, DNS:1.2.3.4, IP Address:10.0.0.1, IP Address:1.2.3.4
+----
+
+The `kubeconfig` file must include one of the server names listed in the {product-title} `config.yaml` file. It must also use the same CA for validation.
+//Q: Does this mean that remote access files are only generated if a `config.yaml' was created?
```

Comment:
We always generate the default kubeconfig files. The names in the `config.yaml` can be used to generate _more_ files.

Reply:
A kubeconfig using the hostname is always generated. Additional ones may be added via configuration. The one for the hostname is also editable to use the node IP instead.

Reply:
@pacevedom If the user edits that one, will the change be saved for the time MicroShift starts?